src/wp-includes/media.php

```diff
@@ -1931,6 +1931,11 @@ function wp_filter_content_tags( $content, $context = null ) {
  * @return string Converted `img` tag with optimization attributes added.
  */
 function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
+	// Images should have source and dimension attributes for the loading optimization attributes to be added.
```

Comment:
The regexes below do check for single quotes and double quotes. Why don't these checks do that? Seems like an oversight.

(And yes I know this is just moved from elsewhere, but now is a great opportunity to discuss this and fix/document it)